app/main/helpers/validation.py

```diff
@@ -238,4 +238,5 @@ class DOS2Validator(G8Validator):
     "g-cloud-8": G8Validator,
     "digital-outcomes-and-specialists": DOSValidator,
     "digital-outcomes-and-specialists-2": DOS2Validator,
+    "g-cloud-9": DOS2Validator,
```

Comment:
Is this a placeholder for the moment or is this our plan to use the DOS2 validator? This reads weird obviously, all the others have appropriate classes for the key, a G9Validator would seem more obvious and consistent?